ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.19.yaml

```diff
@@ -0,0 +1,104 @@
+base_images:
+  ansible:
+    name: "4.19"
+    namespace: ocp
+    tag: ansible
+  aws-efs-csi-operator-create-efs:
+    name: "4.19"
+    namespace: ocp
+    tag: aws-efs-csi-operator-create-efs
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.19"
+    namespace: ocp
+    tag: openstack-installer
+  upi-installer:
+    name: "4.19"
+    namespace: ocp
+    tag: upi-installer
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.19"
+  multi-latest:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.19"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: installer-rehearse-ibmcloud-ipi-sshkey-capi
```

Comment:
Pls remove these capi ci jobs, generally we only keep 2 debug jobs to help easy debug. One is `installer-rehearse-ibmcloud`, another is `installer-rehearse-ibmcloud-dis`

Reply:
enabled the CAPI to debug the known issues, is it OK?

Reply:
The job name in this yaml file is just a placeholder, you can change its content as your desire, but we do not need so many placeholder.